docs/Including_additional_Python_modules.rst

```diff
@@ -0,0 +1,154 @@
+.. _including_additional_python_modules:
+
+Including additional Python modules (``--include-*``)
+=====================================================
+
+EasyBuild's capabilities can be extended easily, by including additional Python modules that implement support for
+building and installing software that is not supported (yet), define additional module naming schemes, or introduce
+additional toolchains, (optionally) with support for additional compilers, MPI libraries, linear algebra libraries, etc.
+
+Since EasyBuild v2.2.0, dedicated configuration options are available that make it straightforward to get EasyBuild to
+pick up additional Python modules, and get them registered in the appropriate ``easybuild`` subnamespace.
+
+
+* :ref:`include_easyblocks`
+* :ref:`include_mns`
+* :ref:`include_toolchains`
+
+
+.. _include_general_remarks:
+
+General aspects of ``--include`` options
```

Comment:
this should maybe be --include-* just to avoid confusion over a generic --include option.